template-compose/data/src/main/java/co/nimblehq/template/compose/data/local/storages/BaseSharedPreferences.kt

```diff
@@ -1,4 +1,4 @@
-package co.nimblehq.template.compose.data.storages
+package co.nimblehq.template.compose.data.local.storages
```

Comment:
Shall we rename this package to `local.preferences`? For example, we may have a local database which may end up in a `local.database` package. "Storage" is typically an uncountable word, so "storages" sounds a bit strange to read 😅 What do you think?

The rest LGTM 🚀

Reply:
@ryan-conway makes sense. At least all components are `Preferences` 🤓 

<img width="236" alt="image" src="https://github.com/nimblehq/android-templates/assets/16315358/6f6b910e-f662-4c9e-b146-a6ab3cafe301">

Fixed in https://github.com/nimblehq/android-templates/pull/536/commits/bde5cf5ced84e6ca22bbf53dc88519fa2a71f7bc